docker-compose.yml

```diff
@@ -2,15 +2,18 @@ version: '2.1'
 volumes:
   core-storage:
   reports-storage:
+  leviathan:
 services:
   core:
     privileged: true
     build:
       context: ./core
-    network_mode: host
+    ports:
+      - "80:80"
```

Comment:
I know this was my suggestion but it seems that host networking is still required for testbot.

I suspect it's because the DUT connects to the Fin via WiFi AP and that gateway isn't guaranteed to be populated in the core container when in bridge networking. So we end up in a state where the worker can reach the device but the core service cannot SSH to it.